app/controllers/coronavirus_controller.rb

```diff
@@ -2,23 +2,64 @@ class CoronavirusController < ApplicationController
   before_action :require_coronavirus_editor_permissions!
   layout "admin_layout"
 
-  CONTENT_ID = "774cee22-d896-44c1-a611-e3109cce8eae".freeze
-  CONTENT_URL = "https://raw.githubusercontent.com/alphagov/govuk-coronavirus-content/master/content/coronavirus_landing_page.yml".freeze
-
   def index; end
 
-  def update_draft
```

Comment:
The `update_draft` message has been renamed `fetch_content_and_push` and is called by two controller actions `update_landing` and `update_business`